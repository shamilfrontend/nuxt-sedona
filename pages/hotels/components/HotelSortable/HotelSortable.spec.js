import { shallowMount } from '@vue/test-utils'
import Component from './index'

describe('HotelSortable', () => {
  let wrapper

  beforeEach(() => {
    wrapper = shallowMount(Component, {
      propsData: {
        count: 10
      }
    })
  })

  test('mounts properly', () => {
    expect(wrapper.isVueInstance()).toBeTruthy()
  })

  test('should match snapshot', () => {
    expect(wrapper.html()).toMatchSnapshot()
  })
})
