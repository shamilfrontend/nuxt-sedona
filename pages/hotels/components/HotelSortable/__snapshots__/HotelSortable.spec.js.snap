// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`HotelSortable should match snapshot 1`] = `
<div class="hotel-sortable"><span class="hotel-sortable__count">Найдено: 10</span>
  <h3 class="hotel-sortable__title">Сортировка:</h3>
  <ul class="hotel-sortable__list">
    <li class="hotel-sortable__item"><a href="#" class="hotel-sortable__link hotel-sortable__link_active">
        По цене
      </a></li>
    <li class="hotel-sortable__item"><a href="#" class="hotel-sortable__link">
        По типу
      </a></li>
    <li class="hotel-sortable__item"><a href="#" class="hotel-sortable__link">
        По рейтингу
      </a></li>
  </ul>
  <div class="sortable-arrows"><a href="#" class="sortable-arrows__link sortable-arrows__up"><svg height="10" viewBox="0 0 11 10" width="11" xmlns="http://www.w3.org/2000/svg" class="sortable-arrows__svg">
        <path d="M5.5 0L0 10h11z" fill="#cacaca"></path>
      </svg></a> <a href="#" class="sortable-arrows__link sortable-arrows__down sortable-arrows_active"><svg height="10" viewBox="0 0 11 10" width="11" xmlns="http://www.w3.org/2000/svg" class="sortable-arrows__svg">
        <path d="M5.5 10L0 0h11" fill="#cacaca"></path>
      </svg></a></div>
</div>
`;
